src/common/types/rsvp.ts

```
@@ -93,42 +40,226 @@ export const rsvpConfigSchema = z
+  "Sustainability in Food & Environmental Systems",
+  "Sustainable Design",
+  "Systems Engineering and Design",
+  "Teaching - Middle Grades Education",
+  "Theatre",
+  "Urban Studies & Planning",
+  "Voice"
+] as const;
+
+export const majorSchema = z.enum(MAJORS).meta({description: "The student's primary major at UIUC"});
+
+export const rsvpProfileSchema = z.object({
```

ESLint

[error] Insert `⏎··`

(prettier/prettier)